Update changes to code post code review feedback.

jrai0792 opened the pull request:
### Sope of the project 
Building an HTML page that matches the appearance of the ### youtube video player page.

### Project Summary
- Added blank index.html file.

- Added navigation bar with its elements.

- Added main video and its elements.

- Added recommended video and elements.

- Added CSS file and styled all the sections.

jrai0792 committed:
Add blank index file

oracleot commented:
I verified the pull request with the added blank index.html file

oracleot committed:
Add title to index page

oracleot committed:
Add elements to the navigation bar

oracleot committed:
Merge pull request #2 from oracleot/navigation-bar-branch

Add elements to the navigation bar

jrai0792 committed:
Add video and its elements.

jrai0792 committed:
Merge pull request #3 from oracleot/video-branch

Add video and its elements.

oracleot committed:
Add recommended video and elements

oracleot committed:
Merge pull request #4 from oracleot/sidebar-recommended-videos

Add recommended video and elements

jrai0792 committed:
Add css style file.

jrai0792 committed:
Merge pull request #5 from oracleot/css-branch

Add css style file.

oracleot committed:
Add style to the header and navigation items

oracleot committed:
Merge pull request #6 from oracleot/css-branch

Add style to the header and navigation items

jrai0792 committed:
Add styles for Main Video

jrai0792 committed:
Merge pull request #7 from oracleot/css-branch

Add styles for Main Video

jrai0792 committed:
Add more style for main video section

jrai0792 committed:
Merge pull request #8 from oracleot/css-branch

Add more style for main video section

oracleot committed:
Add todos to gitignore fo development

oracleot committed:
Update style for search bar in navigation

oracleot committed:
Merge pull request #9 from oracleot/css-branch

Update style for search bar in navigation

oracleot committed:
Update navigation icons to use list

oracleot committed:
Merge pull request #10 from oracleot/css-branch

Update navigation icons to use list

jrai0792 committed:
Add comment section in the main section.

jrai0792 committed:
Merge pull request #11 from oracleot/css-branch

Add comment section in the main section.

jrai0792 committed:
Update icons whereever images where used instead.

jrai0792 committed:
Merge pull request #12 from oracleot/css-branch

Update icons whereever images where used instead.

oracleot committed:
Add styles to recommended videos

oracleot committed:
Merge pull request #13 from oracleot/css-branch

Add styles to recommended videos

jrai0792 committed:
Add styles for comments section

jrai0792 committed:
Merge pull request #14 from oracleot/css-branch

Add styles for comments section

jrai0792 committed:
Rectify HTML errors.

jrai0792 committed:
Merge pull request #15 from oracleot/css-branch

Rectify HTML errors.

oracleot committed:
Update style for the Up Next video

oracleot committed:
Merge pull request #16 from oracleot/css-branch

Update style for the Up Next video

oracleot committed:
Add position fixed to navigation

oracleot committed:
Merge pull request #18 from oracleot/css-branch

Add position fixed to navigation

oracleot committed:
Updated README using Microverse template

HADDADSOHAIB commented on index.html:
```diff
@@ -0,0 +1,249 @@
+          <i class="fa fa-thumbs-o-down" aria-hidden="true"></i>&ensp;4&emsp;
+        </li>
+        <li>
+          <i class="fa fa-share" aria-hidden="true"></i>&ensp;Share&emsp;
```
The share icon should be a button

oracleot replied:
Thanks for the code review, we will resolve ASAP and ask for another review

jrai0792 replied:
Code revamped based on the feedback from the code review.

HADDADSOHAIB reviewed (changes requested):
### Status: Change required :recycle: 

### Hi @jrai0792 and @oracleot :hand: 

### Great job with the project, you did great work. There is some changes to be made to approve the project:
- check my comments for additional requirements.
- In the project description, there was a link that you should have used to build your project:
![image](https://user-images.githubusercontent.com/26064042/87140678-cf2f7480-c299-11ea-8a7f-90c8b7b99024.png)
http://archive.vn/Bss88
there a slight different between your project and that link in the section under the main video.

 | Source Page	| Clone Page |
|:---:|:---:|
|![Screenshot from 2020-07-10 11-51-12](https://user-images.githubusercontent.com/26064042/87147021-cd6aae80-c2a3-11ea-80bf-744427897f73.png) |![Screenshot from 2020-07-10 11-50-59](https://user-images.githubusercontent.com/26064042/87147036-d22f6280-c2a3-11ea-9dea-c69eb0bbb4e7.png) |

- A pull request description should have information about what this pull request add to the project.
![Screenshot from 2020-07-10 12-01-48](https://user-images.githubusercontent.com/26064042/87147841-428ab380-c2a5-11ea-9922-3ae9c169ed56.png)
Please improve the current description

### Once done with the changes, ask for another code review.

jrai0792 committed:
Fixed code reviews comments

peterrobert reviewed (changes requested):
### CHANGES REQIRED :recycle: :stop_button: 
### HELLO @jrai0792 AND @oracleot AM PETER ROBERT A TSE AND I HAVE REVIEWED YOUR PROJECT. 
### YOUR PROJECT LOOKS AMAZING :clap:  VERY NICE WORK :partying_face: HOWEVER, THERE ARE A FEW THINGS THAT HAVE PREVENTED ME FROM APPROVING THIS PROJECT KINDLY CHECK BELOW :arrow_down: 

- [x] Just like stated above you need to make sure you have a descriptive pull request message. You can do this by adding a title and a summary of what you have worked on using bullet points.

- [x] Also you do have a readme file :clap: but let us make it more descriptive by adding more information to it. For example, you can add a screenshot of your project.

### when you are done improving on this kindly submit your project for another review. 
### good work working on the changes that were requested by the previous TSE :clap: 

### HAPPY CODING :computer:

oracleot committed:
Add screenshot for project

- Took a screenshot of project screen from browser
- Uploaded screenshot to GitHub repo to be added to README file

oracleot committed:
Update README to include screenshot

Added a screenshot to the README

oracleot committed:
Update README.md

oracleot committed:
Update README.md

oracleot committed:
Update README.md

peterrobert reviewed (approved):
### APPROVED :+1: :1st_place_medal: 
### VERY IMPRESSIVE WORK YOUR PROJECT IS NOW APPROVED .. GOOD WORK IMPROVING AS REQUESTED :clap: 

[OPTIONAL]
your HTML is showing some warnings when validated by this  [tool](https://validator.w3.org/nu/#textarea) you guys can try and fix the warnings. However, since they are not errors this is not a requirement

### HAPPY CODING :computer:

oracleot commented:
Thanks @peterrobert!
